docs/current/quickstart/319191-quickstart-basics.mdx

```diff
@@ -36,9 +36,8 @@ subgraph program["Your program"]
   lib["Dagger SDK"]
 end
 
-engine["Dagger Engine"]
+engine["Dagger Engine - OCI container runtime"]
```

Comment:
Could you please make this diagram into a partial? We can then also apply it in the index page: https://docs.dagger.io/

Reply:
Done. FWIW: https://github.com/dagger/dagger/pull/4532/commits/572ee79b69d0e27ccef5ab43d78bcb3839cbf90c